FrameworkSystem/DB/ProxyDB.py

```diff
@@ -395,28 +430,34 @@ def storeProxy(self, userDN, userGroup, chain):
     pemChain = chain.dumpAllToString()['Value']
     dValues = {'UserName': self._escapeString(userName)['Value'],
                'UserDN': sUserDN,
-               'UserGroup': sUserGroup,
                'Pem': self._escapeString(pemChain)['Value'],
-               'ExpirationTime': 'TIMESTAMPADD( SECOND, %d, UTC_TIMESTAMP() )' % int(remainingSecs),
-               'PersistentFlag': "'False'"}
+               'ExpirationTime': 'TIMESTAMPADD( SECOND, %d, UTC_TIMESTAMP() )' % int(remainingSecs)}
+    if userGroup and not proxyProvider:
+      dValues['UserGroup'] = sUserGroup
+      dValues['PersistentFlag'] = "'False'"
+    else:
+      dValues['ProxyProvider'] = "'%s'" % proxyProvider
```

Comment:
`dValues['PersistentFlag']` shouldn't be set in this case?

Reply:
In this case table store proxy generated from scratch. I think, it do not allow MyProxy to be used in the near future for cleanProxies table